conf/broker.conf

```diff
@@ -627,6 +627,10 @@ zookeeperSessionExpiredPolicy=reconnect
 # Enable or disable system topic
 systemTopicEnabled=true
 
+# Enable strict topic name check. See PIP-242 to get more information.
+# NOTE: This flag will be removed in the next major release.
```

Comment:
Maybe we can use an exact release version, such as (4.0.0).

Reply:
We can't give user a promise. Because it's OSS, we don't know what will happen.

Reply:
Fixed.